native/src/transportRouteResultSegment.h

```diff
@@ -0,0 +1,133 @@
+#ifndef _OSMAND_TRANSPORT_ROUTE_RESULT_SEGMENT_H
+#define _OSMAND_TRANSPORT_ROUTE_RESULT_SEGMENT_H
+#include "transportRoutingObjects.h"
+
+#define GEOMETRY_WAY_ID -1
+#define STOPS_WAY_ID -2
+
+struct TransportRouteResultSegment {
+    private:
+        static const bool DISPLAY_FULL_SEGMENT_ROUTE = false;
+        static const int  DISPLAY_SEGMENT_IND = 0;
+
+    public:
+    
+        TransportRouteResultSegment() {
+        }
+    
+        SHARED_PTR<TransportRoute> route;
```

Comment:
really needed?

Reply:
seems like yes. Routes, Stops - used in several objects.